app/services/nytimes_service.rb

```diff
@@ -10,6 +10,11 @@ def articles
     response_body = parse(connection.get)
     articles_data = response_body["results"]
     Article.create_articles(articles_data)
+    keyword_engine = KeywordEngine.new
+    Article.all.each do |article|
+      keywords = keyword_engine.generate_keywords(article)
+      article.update(keywords: keywords)
```

Comment:
I think it would probably be better for the article to know about the keyword engine and not the nytimes_service.  I'll update that.